conseil-api/src/main/scala/tech/cryptonomic/conseil/api/ConseilApi.scala

```diff
@@ -134,14 +147,29 @@ class ConseilApi(config: CombinedConfiguration)(implicit system: ActorSystem) ex
         case (key, value) => key.name -> value.dataEndpoint(service)
       }
 
+    private val visiblePlatforms =
+      transformation.overridePlatforms(config.platforms.getPlatforms)
+
     private def forVisiblePlatforms(f: BlockchainPlatform => Api): Map[BlockchainPlatform, Api] =
-      transformation
-        .overridePlatforms(config.platforms.getPlatforms)
-        .map { platform =>
-          val blockchainPlatform = BlockchainPlatform.fromString(platform.name)
-          blockchainPlatform -> f(blockchainPlatform)
-        }
-        .toMap
+      visiblePlatforms.map { platform =>
+        val blockchainPlatform = BlockchainPlatform.fromString(platform.name)
+        blockchainPlatform -> f(blockchainPlatform)
```

Comment:
I don't remember, might've asked before... can we find a meaning for this `f` function?

Like, it's not abstract or generic, so I'd personally appreciate if it could have a proper name explaining what the function's supposed to get you.

Thanks

Reply:
I've changed that to `init` but I've added Scala doc for entire function, which should clarify that a bit.